test/parallel/test-dgram-send-default-host.js

```diff
@@ -16,7 +16,7 @@ const received = [];
 client.on('listening', common.mustCall(() => {
   const port = client.address().port;
   client.send(toSend[0], 0, toSend[0].length, port);
-  client.send(toSend[1], port);
+  client.send(new Uint8Array([...toSend[1]]), port);
```

Comment:
@joyeecheung Right, thanks for pointing that out. I’ve updated the file so all the tests are run twice now, once with the original input types and once with Uint8Arrays.